docs/src/man/seq.md

```diff
@@ -91,6 +91,45 @@ true
 
 ```
 
+Every nucleotide is encoded using lower 4 bits of a byte. An unambiguous
+nucleotide has only one *true* bit and the other bits are *false*s. The table
```

Comment:
"using lower 4 bits" <- "using the lower 4 bits".

Also should it be "has only one set bit" & "the other bits are unset"? I'm not sure which is more intuitive to less computer savvy biologists,